web/src/composebox_typeahead.ts

```diff
@@ -1013,7 +1265,8 @@ export function content_typeahead_selected(item, query, input_element, event) {
                     rest = rest.slice(1);
                 }
                 $textbox.val(beginning + rest);
-                $textbox.caret(beginning.length, beginning.length);
+                // TODO(prep) commit for removing unecessary argument
+                $textbox.caret(beginning.length);
```

Comment:
`caret` only takes one argument, I'll make a prep commit removing the second argument here and below